Problems/2750/ans_2750.py

```diff
@@ -0,0 +1,17 @@
+#https://www.acmicpc.net/problem/2750
+
+def solve():
+    num_size = int(input())
+    num_list = []
+    for i in range(num_size):
+        num_list.append(input())
+    num_list = list(map(int, num_list))
```

Comment:
꼭 list로 변경할 필요 없이 int(input())을 통해 type 변환을 하지 않아도 될 것으로 보입니다.

```suggestion
        num_list.append(int(input()))
```